index.html

```diff
@@ -0,0 +1,253 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div class="container w-50 p-5">
+      <header class="header">
+        <h1 id="title" class="text-center">Our Survey Form</h1>
+
```

Comment:
It would be super helpful if you always give an explanation for your requirements. One of the many objectives code reviews have is to help the person being reviewed, and what better way to accomplish this than to give them a clear reason on why we are suggesting the changes 🤓 Kindly explain in your comments the reason for your changes 😎

Reply:
Please be sure to tell your requirements/suggestions in a way that it sounds like suggestions instead of orders 😉 The idea is to make it politely and friendly 😬